svcs/guacamole/intercept.go

```diff
@@ -14,216 +14,118 @@ import (
 )
 const waitingHTMLTemplate = `
```

Comment:
I think this direction is also conceptually not ideal. Assigning labs to existing events (on boot) should ideally just wait with giving users access to the frontend until every known team has assigned a lab. In the case of new events, people will be more-or-less instantly assigned a lab or should be notified that labs are either maxed out or unavailable during signup.

Reply:
Yes this may be a better way than queuing the request, at least a lot simpler. 

However it should be handled a bit better in the frontend, as an error will undo all the information in the registration form. 
I imagine this is a lot easier then the amigo frontend is done.